src/audit.ts

```diff
@@ -124,6 +126,35 @@ export default class Audit {
         await pipeline.exec();
     }
 
+    async saveUnsetReconfigureEvents(instanceId: string, group: string): Promise<void> {
+        const pipeline = this.redisClient.pipeline();
+        const value: InstanceAudit = {
+            instanceId: instanceId,
+            type: 'reconfigure-complete',
+            timestamp: Date.now(),
+        };
+        pipeline.set(`audit:${group}:${instanceId}:reconfigure-complete`, JSON.stringify(value), 'ex', this.auditTTL);
+        await pipeline.exec();
+    }
+
+    async saveReconfigureEvents(instanceDetails: Array<InstanceDetails>): Promise<void> {
```

Comment:
We should probably add inside the audit's `saveLatestStatus` a call to `increaseReconfigureEventExpiration`, similar to `increaseShutdownEventExpiration`. The purpose is to keep the reconfigure audit in redis as long as the instance is still active.

Reply:
agreed